Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/fubar/ubuntu-indexer
```

Comment:
Right. In the design taking shape in my mind, this repo would get DXR to the point where it's able to index code of all languages it supports. (So, for example, rust would be on the image.) Ideally, we could add the dev-specific stuff (Sphinx, nose, …) as a separate layer or make target or env var switch (whatever makes sense) so it's suitable both for dev and to base deployment images on.